.github/workflows/run-nightly-e2e.yaml

```diff
@@ -4,6 +4,9 @@ on:
   schedule:
     - cron:  '22 2 * * *'
 
+  # Manual trigger to allow for debugging
+  workflow_dispatch:
```

Comment:
So this gets defined without any values?

Reply:
yep, you don't need to add any value to trigger it, and the workflow doesn't need any value as well.